schemas/past_officers.schema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://ocf.berkeley.edu/schemas/past_officers.schema.json",
+  "type": "object",
+  "properties": {
+    "terms": {
+      "type": "array",
+      "items": {
+        "$ref": "#/$defs/term"
+      }
+    }
+  },
+  "required": [
+    "terms"
+  ],
+  "$defs": {
+    "term": {
+      "type": "object",
+      "required": [
+        "termSemester",
+        "termOfficers"
+      ],
+      "properties": {
+        "termSemester": {
+          "type": "string"
+        },
+        "termOfficers": {
+          "type": "object",
+          "required": [
+            "gms",
+            "sms"
+          ],
+          "properties": {
+            "gms":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
+                }
+            },
+            "sms":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
+                }
+            },
+            "gmsDeputies":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
+                }
+            },
+            "smsDeputies":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
+                }
+            },
+            "committeeHeads":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
```

Comment:
This doesn't say which committee they're head of. You're including that in `pastOfficer`, which is awkward because it's not a meaningful property to add to the SM/GM/deputies lists. It also means the committee names are getting repeated more than necessary.

This should probably be an object that looks something like:

```yaml
committeeHeads:
  "Internal":
    - handle: jaw
    - handle: ronitnath
...
```

It can also be just a list of strings if we don't expect this to be as complicated as SMs with start/end dates.